st2actions/st2actions/notifier/notifier.py

```diff
@@ -159,6 +181,22 @@ def _post_notify_subsection_triggers(self, liveaction=None, execution_id=None,
             if len(failed_routes) > 0:
                 raise Exception('Failed notifications to routes: %s' % ', '.join(failed_routes))
 
+    def _build_jinja_context(self, liveaction, execution):
+        context = {SYSTEM_KV_PREFIX: KeyValueLookup()}
+        context.update(liveaction.parameters)
```

Comment:
I supposed we could add a prefix in ActionChain as well. It sounds like a mistake due to inexperience in ActionChain. I do not think we should continue with the same pattern here as well.